src/app/data/data.json

```
@@ -0,0 +1,528 @@
+{
+    "BtcPayServer": {
+        "name": "BtcPayServer",
+        "description": "",
+        "link": "https://github.com/btcpayserver/btcpayserver/issues",
+        "org": {
+            "link": "",
+            "name": ""
+        },
+        "image": "",
+        "tags": [],
+        "metadata": {
+            "twitter": "",
+            "website": ""
+        },
+        "languages": ["C#", "Shell"]
+    },
+
+    "Wasabi": {
+        "name": "Wasabi",
+        "description": "",
+        "link": "https://github.com/zkSNACKs/WalletWasabi/issues",
+        "org": {
+            "link": "",
+            "name": ""
+        },
+        "image": "",
+        "tags": [],
+        "metadata": {
+            "twitter": "",
+            "website": ""
+        },
+        "languages": ["C#"]
+    },
+
+    "Fedimint": {
+        "name": "Fedimint",
+        "description": "",
+        "link": "https://github.com/fedimint/fedimint/issues",
+        "org": {
+            "link": "",
+            "name": ""
+        },
+        "image": "",
+        "tags": [],
+        "metadata": {
+            "twitter": "",
+            "website": ""
+        },
+        "languages": ["Rust", "Shell"]
+    },
+
+    "Polar": {
+        "name": "Polar",
+        "description": "",
+        "link": "https://github.com/jamaljsr/polar/issues",
+        "org": {
+            "link": "",
+            "name": ""
+        },
+        "image": "",
+        "tags": [],
+        "metadata": {
+            "twitter": "",
+            "website": ""
+        },
+        "languages": ["Typescript", "Docker"]
```

Lets stick with one convention of writing the languages. Also, Docker is not a language
```suggestion
        "languages": ["TS"]
```

---

Alright